### PR TITLE
add the ability to edit user's data

### PR DESCRIPTION
**Pull request by natalya5**
code review:
- [x] @CHILL99 
- [x] @krisberry 
- [x] @DimaSmagulov 
- [x] @oleg08 
- [x] @OrestF 
- [ ] @rtershak 
- [x] @vvs79 

2nd lvl review:

- [ ] @Kmalynss
- [x] @serhiym 
- [ ] @ruba-ruba 

https://trello.com/c/tZfDZkop/40-as-registered-user-i-want-to-be-able-edit-my-personal-data-and-reset-password

### CODE REVIEW

File: app/views/layouts/_header.html.slim [hunk 1]
```diff
@@ -9,11 +9,19 @@ nav.navbar.navbar-default.navbar-fixed-top.topnav[role="navigation"]
           = link_to "Home", authenticated_root_path
         li
           a[href="#help"]
-            | Help
+            | Help
```
Comment (serhiym) [inline app/views/layouts/_header.html.slim]: remove space after "Help"

File: app/views/layouts/_header.html.slim [hunk 2]
```diff
@@ -9,11 +9,19 @@ nav.navbar.navbar-default.navbar-fixed-top.topnav[role="navigation"]
...
+                = link_to('Profile', edit_user_registration_path)
+              li.divider
+              li
+                = link_to('Logout', destroy_user_session_path, :method => :delete)
         - else
           li
-            = link_to('Login', new_user_session_path)
-        
\ No newline at end of file
+            = link_to('Login', new_user_session_path)
```
Comment (serhiym) [inline app/views/layouts/_header.html.slim]: remove spaces and add new line to the end of file

**Comment by serhiym** [comment]
lgtm

**Comment by vvs79** [comment]
reviewed

**Comment by DimaSmagulov** [comment]
Sorry but i can't put a tic)) reviewed

**Comment by krisberry** [comment]
reviewed

**Comment by oleg08** [comment]
reviewed

**Comment by CHILL99** [comment]
reviewed

**Comment by OrestF** [comment]
Reviewed

**Comment by serhiym** [comment]
@natalya5 - please remove spaces and let's  merge it

**Comment by natalya5** [comment]
spaces removed

**Pull request merged**